src/Compilers/CSharp/Test/Syntax/Parsing/UTF8StringLiteralsParsingTests.cs

```diff
@@ -296,5 +296,47 @@ public void Errors_12()
             }
             EOF();
         }
+
+        [Fact]
+        public void Interpolation_01()
+        {
+            UsingExpression(@"$""hello""u8",
```

Comment:
It might be good to modify the proposal to specify what happens when `@` or `$` prefix is used, and either:
- the u8 suffix is used, or
- the new utf8 conversion is used

Intuitively I thought that a constant-valued interpolated string would be fine to use with the u8 suffix, but that doesn't appear to be the case. I don't have a strong position on that, but it seems like whatever is decided should be included in the proposal.

Reply:
I am not sure where confusion is coming from. I think the proposal is clear: "the language will provide the `u8` suffix **on string literals** to force the type to be UTF8." The suffix is for literals, not for constant expressions. Interpolated strings are not literals, even when they have a constant value.